src/visitor.cpp

```diff
@@ -41,6 +41,23 @@ namespace
 
         return cb(functor, container, {visitor_info::container_entity_exit, last_child});
     }
+
+    template <typename T>
+    bool handle_container(const cpp_entity& e, detail::visitor_callback_t cb, void* functor,
+                          detail::visitor_filter_t fcb, void* filter,
+                          bool last_child)
+    {
+        auto& container = static_cast<const T&>(e);
+
+        for (auto iter = container.begin(); iter != container.end();)
+        {
+            auto& cur = *iter;
+            ++iter;
+            detail::visit(cur, fcb, filter, cb, functor, iter == container.end());
+        }
+
+        return fcb(filter, e) && cb(functor, container, {visitor_info::container_entity_exit, last_child});
```

Comment:
Yep, still WIP (Github is a bit confusing, because it added the early comments after my pull request post where I wrote that).